RUFAS/routines/manure/manure_treatments/anaerobic_lagoon.py

```diff
@@ -115,6 +115,12 @@ def _daily_update_helper(self) -> ManureTreatmentDailyOutput:
         methane_loss, methane_emission_from_degradable_volatile_solids = self._update_methane_emission(
             self._accumulated_output
         )
+
+        if self.config.manure_cover == "cover and flare":
```

Comment:
```suggestion
        if self.config.manure_cover == ManureCoverEnum.COVER_AND_FLARE.value:
```
Ideally, `self.config.manure_cover` would be an enum element and not a string, and would be compared directly to the `ManureCoverEnum`.